doc/commands.dox

```diff
@@ -2066,9 +2066,9 @@ ALIASES  = "english=\if english" \
   present in the function declaration or definition.
 
   The \c \\param command has an optional attribute, \<dir\>, specifying the direction
-  of the parameter. Possible values are "[in]", "[in,out]", and "[out]",
-  note the [square] brackets in this description. Note that it is also possible to put
-  whitespace between the command and the \<dir\>.
+  of the parameter. Possible values are "[in]", "[out]", "[inout]", "[outin]",
+  "[in,out]", and "[in out]"; note the [square] brackets in this description.
+  Note that it is also possible to put whitespace between the command and the \<dir\>.
```

Comment:
You are a bit faster than I was (I was completing another issue) but also prepared a patch with the text:
```
  The \c \\param command has an optional attribute, \<dir\>, specifying the direction
  of the parameter. Possible values are "[in]", "[in,out]", "[out,in]", and "[out]",
  note the [square] brackets in this description. Also supported are "[inout]", "[outin]"
  as well as the situation where the comma (`,`) is replaced by blanks.
  Note that it is also possible to put whitespace between the command and the \<dir\>.
```
your patch is missing the, uncommon, `"[out in]"` possibility

Reply:
> You are a bit faster than I was (I was completing another issue) but also prepared a patch with the text:

I though I might as well :).

> your patch is missing the, uncommon, "[out in]" possibility

I've applied the missing ones. If you prefer we can go in your direction instead of specifying the all explicitly; I'll do whatever you prefer!

Reply:
Best is to distill the best of both.